src/heitech.zer0mqXt.core/Patterns/PubSub.cs

```diff
@@ -93,15 +113,17 @@ private XtResult RegisterHandler<TMessage>(Action<TMessage> syncCallback = null,
                     asyncCallback: asyncCallback
                 );
 
-                var (success, exception) = next.Setup();
+                var (success, ex2) = next.Setup();
```

Comment:
I can't wait for C# 10, with which we can have both declarations and assignments in the same deconstruction 😉

Reply:
Yeah I was under the impression that it already was able to do that, boi was I in for a surprise :D